### ISSUE TITLE
Bug in the 3D add-on on the LiveTest version

### ISSUE THREAD

**Issue by Goshanchick** [PenguinMod/PenguinMod-Packager#10]
I have prepared a 3D model in a file .fbx and decided to add it to the project. But there is one problem - to add a model, you need to make it under the data uri protocol, but some characters are not encoded in base64, such as a percentage with two circles and so on. How to solve it?

**Comment by JeremyGamer13** [comment]
if you import the file as a data URL using the files extension, does it work?

**Comment by Goshanchick** [comment]
No, because the "Files" extension can only load data from a uri but unfortunately it is impossible to convert them to a uri. What to do?
